docs/userGuide/contentAuthoring.md

```diff
@@ -125,6 +125,29 @@ You can insert variables to be replaced by pre-defined values anywhere in your s
 
 Some helpful variables are provided by MarkBind for your convenience, as shown below. 
 
+#### Font Awesome Icons
+
+This asset is provided by [Font Awesome](https://fontawesome.com/) under their [free license](https://fontawesome.com/license).
+<br><br>
+<tip-box>
+Traditional usage:  
+`<span class='fas fa-chess-knight'></span>`
+<br>  
+MarkBind usage:  
+<code>{<span></span>{fas_chess_knight}}</code>
+</tip-box>
+
+Font Awesome has three different styles for their free icons, each with their own prefix:
+<strong>Solid</strong> (`fas`), <strong>Regular</strong> (`far`) and <strong>Brands</strong> (`fab`).<br>
+Be sure to check Font Awesome's [full list of icons](https://fontawesome.com/icons?d=gallery&m=free) on which prefix to use.<br>

```

Comment:
"on which prefix to use." Maybe should just say for "the full name of the icon" or similar?

Also it can be easily confused by what we meant by prefix --> "fas " and "fa-" are both techincally prefixes.

Reply:
The message I was trying to convey was that some icons do not have certain styles.

Maybe change to `"... full list of icons and which styles they support."`